dist/assests/css/header-banner-nav.css

```diff
@@ -0,0 +1,161 @@
+@import url('header-banner-nav.css');
+/* Main Section, Header, Nav */
+.main-header {
+  background-image: url('../img/banner.jpg');
+  height: 100vh;
+  background-repeat: no-repeat;
+  background-size: cover;
+  position: relative;
+  width: 100%;
+}
+
+.main-header .main-header__nav {
+  /* background-color: blue; */
+  display: flex;
+  justify-content: space-between;
+  align-items: baseline;
+  width: 100%;
+  margin-right: 1rem;
+  font-size: 11px;
+  font-weight: 600;
+  letter-spacing: 0.02em;
+  position: absolute;
+  top: 0;
+  left: 0;
+  z-index: 40;
+}
+.main-header__nav .left {
```

Comment:
`.left` is not descriptive, better to name it something along the lines `main-navigation` or `left-navigation`